index.js

```diff
@@ -50,7 +52,16 @@ function testGenResponse(swagger, path, operation, response, config) {
       description: (response + ' ' +
         swagger.paths[path][operation].responses[response].description),
       assertion: config.assertionFormat,
-      parameters: [],
+      bodyParameters: parametersArray.bodyParameters,
+      queryParameters: parametersArray.queryParameters,
+      headerParameters: parametersArray.headerParameters,
+      pathParameters: parametersArray.pathParameters,
+      formParameters: parametersArray.formParameters,
+      contentType: swagger.consumes ===
+      undefined ? 'application/json' : swagger.consumes[0],
```

Comment:
What if `swagger.consumes` is an empty array? We should guard against that. Same for `produces`